.github/workflows/mirror-certs.yaml

```diff
@@ -12,8 +12,9 @@ jobs:
       - uses: actions/checkout@v2
       - name: Download certificates
         run: |
```

Comment:
Did some research on command for learning purposes:

-r recursive
-np no-parent (do not ascend)
-no-host-directories (disable generate host-prefix directories)
--cut-dirs 5 (Ignore number directory components)
-A.cer (Specify comma-separated lists of file name suffixes or patterns to accept)
-P prefix ( the directory where all other files and subdirectories will be saved)


Not sure how `${ANCHORS}` comes into play, but would be nice